TRB_label_assist/classification_performance.ipynb

```diff
@@ -49,10 +49,11 @@
                 "labeled_trip_df_map = {}\n",
                 "expanded_labeled_trip_df_map = {}\n",
                 "expanded_all_trip_df_map = {}\n",
+                "ct_entry={}\n",
                 "for u in all_users:\n",
                 "    ts = esta.TimeSeries.get_time_series(u)\n",
-                "    ct_df = ts.get_data_df(\"analysis/confirmed_trip\")\n",
-                "\n",
+                "    ct_entry[u]=eamtr._get_training_data(u,None)\n",
+                "    ct_df = ts.to_data_df(\"analysis/confirmed_trip\",itertools.chain(ct_entry[u]))\n",
```

Comment:
Ditto

Reply:
we do not need this. Following the code trail, I felt it would be needed, but the code runs fine without. REMOVED